composer.json

```diff
@@ -10,77 +10,97 @@
             "src/Kernel.php"
         ]
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/OpenConext/Stepup-Middleware-clientbundle"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/OpenConext/Stepup-saml-bundle"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/OpenConext/Stepup-bundle"
+        }
+    ],
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.2|^8.0",
+        "php": "^8.2",
+        "ext-gd": "*",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "^6",
+        "ext-mbstring": "*",
+        "ext-openssl": "*",
+        "doctrine/annotations": "^2.0",
+        "guzzlehttp/guzzle": "^7",
         "incenteev/composer-parameter-handler": "~2.0",
-        "jms/translation-bundle": "^1.3.0",
-        "mpdf/mpdf": "^7.0",
-        "nelmio/security-bundle": "^2",
+        "jms/translation-bundle": "^2.0.0",
+        "mpdf/mpdf": "^8.2",
+        "nelmio/security-bundle": "^3",
         "openconext/monitor-bundle": "^3.1",
-        "sensio/framework-extra-bundle": "^5.0",
-        "surfnet/stepup-bundle": "^5.0",
-        "surfnet/stepup-middleware-client-bundle": "^4.1",
-        "symfony/console": "4.4.*",
-        "symfony/expression-language": "4.4.*",
-        "symfony/flex": "^1.8",
-        "symfony/form": "4.4.*",
-        "symfony/http-foundation": "^5.0",
+        "surfnet/stepup-bundle": "dev-feature/sf6-upgrade as 6.0.6",
+        "surfnet/stepup-middleware-client-bundle": "dev-feature/sf6-support",
+        "surfnet/stepup-saml-bundle": "dev-feature/kill-off-travis as 6.0.8",
+        "symfony/console": "^6.3",
```

Comment:
Even tho ^6.3 includes the entire 6.4 tier, I'd much prefer we set the constraint to be `^6.4`. Ofc goes for all SF deps